applications/tari_console_wallet/src/ui/components/balance.rs

```diff
@@ -46,15 +46,12 @@ impl<B: Backend> Component<B> for Balance {
         f.render_widget(block, area);
 
         let balance = app_state.get_balance();
-
+        let time_locked = balance.time_locked_balance.unwrap_or_else(|| MicroTari::from(0u64));
         let available_balance = Spans::from(vec![
             Span::styled("Available:", Style::default().fg(Color::Magenta)),
```

Comment:
Is it worth changing "Available" to "Spendable" ?

Reply:
I think it makes logical sense like this. The only reason we feel the need to differentiate is our context of how it's represented in the backend.